src/FakeItEasy.dotnet/project.json

```diff
@@ -18,8 +18,6 @@
   },
 
   "frameworks": {
-    "netstandard1.6": {
-      "imports": [ "dotnet5.4" ]
-    }
+    "netstandard1.6": { }

```

Comment:
dotnet5.4 is the old TFM that's replaced by netstandard1.6.  Some packages targeting dotnet5.* haven't moved to the netstandard TFM so `imports` make them available.  However, `imports` can be problematic if imported TFM is incompatible.